scripts/create-kubestellar-demo-env.sh

```diff
@@ -139,9 +104,83 @@ checking_cluster() {
 
     done
 }
+##########################################
+
+echo -e "\nStarting environment clean up..."
+echo -e "Starting cluster clean up..."
+
```

Comment:
I would include https://github.com/kubestellar/kubestellar/blob/b68baf3a590729e52ab3f1c4fc56efd9bf381068/docs/content/direct/get-started.md?plain=1#L35 here, for the same reason that it is there.

Reply:
included